src/platform/plugins/shared/data/server/search/session/session_service.ts

```diff
@@ -191,9 +196,13 @@ export class SearchSessionService implements ISearchSessionService {
     const sessionStatuses = await Promise.all(
       findResponse.saved_objects.map(async (so) => {
         const sessionStatus = await getSessionStatus(
-          { internalClient: internalElasticsearchClient },
+          {
+            internalClient: internalElasticsearchClient,
+            asUserClient: asCurrentUserElasticsearchClient,
+          },
           so.attributes,
-          this.sessionConfig
+          this.sessionConfig,
+          this.logger
```

Comment:
all here is necessary for ES|QL support, the logger stuff is optional

FYI @AlexGPlay @lukasolson